test-e2e/project-export.js

```diff
@@ -0,0 +1,361 @@
+import test from 'node:test'
+import assert from 'node:assert/strict'
+import { KeyManager } from '@mapeo/crypto'
+import RAM from 'random-access-memory'
+import Fastify from 'fastify'
+import * as b4a from 'b4a'
+import { generate } from '@mapeo/mock-data'
+import { valueOf } from '@comapeo/schema'
+import { temporaryDirectoryTask } from 'tempy'
+import StreamZip from 'node-stream-zip'
+import { join } from 'node:path'
+import { fileURLToPath } from 'node:url'
+import { createReadStream } from 'node:fs'
+
+import { MapeoManager } from '../src/mapeo-manager.js'
+import { kGeoJSONFileName, kExportGeoJSONStream } from '../src/mapeo-project.js'
+import { blobMetadata } from '../test//helpers/blob-store.js'
+
+/** @import { Readable } from 'streamx' */
+
+const DEFAULT_OBSERVATIONS = 2
+const DEFAULT_TRACKS = 2
+const OBSERVATIONS_PER_TRACK = 2
+
+const BLOB_FIXTURES = fileURLToPath(
+  new URL('../test/fixtures/blob-api/', import.meta.url)
+)
+
+test('Project export empty GeoJSON to stream', async () => {
+  const manager = await setupManager()
```

Comment:
no await needed here (and for other tests in this file using same setup)
```suggestion
  const manager = setupManager()
```